programs/server/config.xml

```diff
@@ -1257,7 +1257,7 @@
         <flush_on_crash>false</flush_on_crash>
     </error_log>
 
-    <!-- Query metric log contains rows Contains history of memory and metric values from table system.events for individual queries, periodically flushed to disk
+    <!-- Query metric log contains history of memory and metric values from table system.events for individual queries, periodically flushed to disk
```

Comment:
This was actually a my copypasterino mistake. Sorry! :pray:

Reply:
Happy to help, really appreciate ClickHouse!